rust/kcl-lib/src/std/mirror.rs

```diff
@@ -89,34 +127,10 @@ async fn inner_mirror_2d(
     // But if you mirror2d a sketch these IDs might change so we need to get the children versus
     // using the IDs we already have.
     // We only do this with mirrors because otherwise it is a waste of a websocket call.
-    for sketch in &mut starting_sketches {
-        let response = args
-            .send_modeling_cmd(
-                exec_state.next_uuid(),
-                ModelingCmd::from(mcmd::EntityGetAllChildUuids { entity_id: sketch.id }),
-            )
-            .await?;
-        let OkWebSocketResponseData::Modeling {
-            modeling_response:
-                OkModelingCmdResponse::EntityGetAllChildUuids(EntityGetAllChildUuids { entity_ids: child_ids }),
-        } = response
-        else {
-            return Err(KclError::Internal(KclErrorDetails::new(
-                "Expected a successful response from EntityGetAllChildUuids".to_string(),
-                vec![args.source_range],
-            )));
-        };
-
-        if child_ids.len() >= 2 {
-            // The first child is the original sketch, the second is the mirrored sketch.
-            let child_id = child_ids[1];
-            sketch.mirror = Some(child_id);
-        } else {
-            return Err(KclError::Type(KclErrorDetails::new(
-                "Expected child uuids to be >= 2".to_string(),
-                vec![args.source_range],
-            )));
-        }
+    for i in 0..starting_sketches.len() {
+        starting_sketches[i].mirror = Some(mirrored_ids[i]);
```

Comment:
I'm not really an expert in this, but it seems like the only thing the `sketch.mirror` field is used for is the `edge_id` argument for `Solid3dGetExtrusionFaceInfo` just after it's extruded. So it really comes down to what makes sense for that.

Based on what we talked about for #7178, this new code makes sense to me. Seems a lot more straight-forward than `EntityGetAllChildUuids` and using the second element. But whether this is needed is kind of beyond my understanding.

If we do go with this, at the top of this function, the `sketch.mirror` fields get filled in with new IDs, and that "Update all to have a mirror" loop probably should get deleted in favor of this. Maybe the exception is mock execution. Need to make sure something gets filled in before any early returns.